_includes/components/mediaelement.html

```diff
@@ -0,0 +1,45 @@
+<figure class="component-mediaelement">
+    <video
+        width="100%" height="340"
+        controls="controls"
+        preload="none"
+        poster="{{relroot}}{{ site.data.videos[include.key].video-preview-image['default'] }}"
+        data-audio-description='[{"src": "{{relroot}}{{ site.data.videos[include.key].mp3-voiceover-src['default'] }}", "type": "audio/mp3"}]'
+        data-audio-description-voiceover="true"
+    >
+        <source type="video/mp4" src="{{relroot}}{{ site.data.videos[include.key].mp4-src['default'] }}" />
+        {% for subtitle in site.data.videos[include.key].subtitles %}
+            <track src="{{ subtitle.src }}" kind="subtitles" srclang="subtitle.lang" label="{{ site.data.languages[subtitle.lang].label }}">
```

Comment:
`srclang` wordt niet goed ingevuld.

```suggestion
            <track src="{{ subtitle.src }}" kind="subtitles" srclang="{{ subtitle.lang }}" label="{{ site.data.languages[subtitle.lang].label }}">
```